2025-03-18 12:46:20,990 [INFO] USER:anand_yadav handlers.py:106:syndicate LOG: Configuration used: C:\Users\anand_yadav\AWS\task11\.syndicate-config-dev
2025-03-18 12:46:21,119 [WARNING] USER:anand_yadav __init__.py:190:initialize_project_state LOG: [93mConfig is set and generated but project state does not exist, seems that you've come from the previous version.[0m
2025-03-18 12:46:21,119 [WARNING] USER:anand_yadav __init__.py:193:initialize_project_state LOG: [93mGenerating project state file (.syndicate) from the existing structure...[0m
2025-03-18 12:46:21,122 [INFO] USER:anand_yadav project_state.py:396:_update_lambdas_from_path LOG: Going to resolve any lambda names from a given path: C:\Users\anand_yadav\AWS\task11\app\lambdas.
2025-03-18 12:46:21,125 [INFO] USER:anand_yadav processor.py:143:_prepare_bucket_view LOG: Viewing complement, URIBucketView, has been found, setting up the raw data.
2025-03-18 12:46:22,417 [INFO] USER:anand_yadav sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-18 12:46:23,745 [INFO] USER:anand_yadav handlers.py:142:test LOG: Running tests...
2025-03-18 12:46:23,746 [INFO] USER:anand_yadav handlers.py:150:test LOG: Tests not found, 'tests' folder is missing in 'C:/Users/anand_yadav/AWS/task11'.
2025-03-18 12:46:23,746 [INFO] USER:anand_yadav helper.py:330:timed LOG: Stage test, elapsed time: 0:00:00.000646
2025-03-18 12:46:23,748 [INFO] USER:anand_yadav handlers.py:993:assemble LOG: Building artifacts, bundle: task11_250318.124622
2025-03-18 12:46:23,749 [INFO] USER:anand_yadav project_state.py:396:_update_lambdas_from_path LOG: Going to resolve any lambda names from a given path: C:\Users\anand_yadav\AWS\task11\jsrc\main\java\com\task11.
2025-03-18 12:46:23,749 [INFO] USER:anand_yadav project_state.py:396:_update_lambdas_from_path LOG: Going to resolve any lambda names from a given path: C:\Users\anand_yadav\AWS\task11\src\lambdas.
2025-03-18 12:46:23,749 [INFO] USER:anand_yadav project_state.py:396:_update_lambdas_from_path LOG: Going to resolve any lambda names from a given path: C:\Users\anand_yadav\AWS\task11\app\lambdas.
2025-03-18 12:46:23,749 [INFO] USER:anand_yadav project_state.py:396:_update_lambdas_from_path LOG: Going to resolve any lambda names from a given path: C:\Users\anand_yadav\AWS\task11\dnapp\lambdas.
2025-03-18 12:46:23,749 [INFO] USER:anand_yadav project_state.py:430:_update_bpm_resources_from_path LOG: Going to resolve any build project mapping dependent resources from a given path: C:\Users\anand_yadav\AWS\task11\swagger_src.
2025-03-18 12:46:23,749 [INFO] USER:anand_yadav project_state.py:430:_update_bpm_resources_from_path LOG: Going to resolve any build project mapping dependent resources from a given path: C:\Users\anand_yadav\AWS\task11\appsync_src.
2025-03-18 12:46:23,749 [INFO] USER:anand_yadav handlers.py:821:assemble_node LOG: Command assemble node: project_path: app 
2025-03-18 12:46:23,750 [INFO] USER:anand_yadav nodejs.py:58:assemble_node_lambdas LOG: Going to package lambdas starting by path C:\Users\anand_yadav\AWS\task11\app
2025-03-18 12:46:23,750 [INFO] USER:anand_yadav handlers.py:830:assemble_node LOG: NodeJS artifacts were prepared successfully.
2025-03-18 12:46:23,750 [INFO] USER:anand_yadav helper.py:330:timed LOG: Stage assemble_node, elapsed time: 0:00:00.001186
2025-03-18 12:46:23,752 [INFO] USER:anand_yadav helper.py:330:timed LOG: Stage assemble, elapsed time: 0:00:00.003242
2025-03-18 12:46:23,753 [INFO] USER:anand_yadav handlers.py:1037:package_meta LOG: Package meta, bundle: task11_250318.124622
2025-03-18 12:46:23,753 [INFO] USER:anand_yadav meta_processor.py:513:create_meta LOG: Bundle path: C:/Users/anand_yadav/AWS/task11/bundles/task11_250318.124622
2025-03-18 12:46:23,754 [INFO] USER:anand_yadav helper.py:292:write_content_to_file LOG: C:\Users\anand_yadav\AWS\task11\.syndicate-config-dev/bundles/task11_250318.124622\build_meta.json file was created.
2025-03-18 12:46:23,755 [INFO] USER:anand_yadav handlers.py:1040:package_meta LOG: Meta was configured successfully.
2025-03-18 12:46:23,755 [INFO] USER:anand_yadav helper.py:330:timed LOG: Stage package_meta, elapsed time: 0:00:00.002235
2025-03-18 12:46:23,915 [INFO] USER:anand_yadav handlers.py:1083:upload LOG: Upload bundle: task11_250318.124622
2025-03-18 12:46:24,235 [INFO] USER:anand_yadav bundle_processor.py:232:upload_bundle_to_s3 LOG: Bundle was validated successfully
2025-03-18 12:46:24,567 [INFO] USER:anand_yadav handlers.py:1090:upload LOG: Bundle was uploaded successfully
2025-03-18 12:46:24,567 [INFO] USER:anand_yadav helper.py:330:timed LOG: Stage upload, elapsed time: 0:00:00.810749
2025-03-18 12:46:24,569 [INFO] USER:anand_yadav helper.py:330:timed LOG: Stage build, elapsed time: 0:00:02.151122
2025-03-18 12:46:28,850 [INFO] USER:anand_yadav handlers.py:106:syndicate LOG: Configuration used: C:\Users\anand_yadav\AWS\task11\.syndicate-config-dev
2025-03-18 12:46:28,998 [INFO] USER:anand_yadav processor.py:143:_prepare_bucket_view LOG: Viewing complement, URIBucketView, has been found, setting up the raw data.
2025-03-18 12:46:30,201 [INFO] USER:anand_yadav project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-18 12:46:30,201 [INFO] USER:anand_yadav project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-18 12:46:30,536 [INFO] USER:anand_yadav sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-18 12:46:30,867 [INFO] USER:anand_yadav project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-18 12:46:30,868 [INFO] USER:anand_yadav project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-18 12:46:31,202 [INFO] USER:anand_yadav sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-18 12:46:31,202 [INFO] USER:anand_yadav helper.py:233:resolve_default_value LOG: Resolved value of deploy_name: task11
2025-03-18 12:46:31,539 [INFO] USER:anand_yadav project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-18 12:46:31,539 [INFO] USER:anand_yadav project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-18 12:46:31,872 [INFO] USER:anand_yadav sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-18 12:46:31,872 [INFO] USER:anand_yadav helper.py:233:resolve_default_value LOG: Resolved value of bundle_name: task11_250318.124622
2025-03-18 12:46:32,214 [INFO] USER:anand_yadav project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-18 12:46:32,214 [INFO] USER:anand_yadav project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-18 12:46:32,571 [INFO] USER:anand_yadav sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-18 12:46:32,914 [INFO] USER:anand_yadav project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-18 12:46:32,914 [INFO] USER:anand_yadav project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-18 12:46:33,248 [INFO] USER:anand_yadav sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-18 12:46:35,083 [INFO] USER:anand_yadav deployment_processor.py:466:create_deployment_resources LOG: Prefixes and suffixes of any resource names have been resolved
2025-03-18 12:46:35,084 [INFO] USER:anand_yadav deployment_processor.py:489:create_deployment_resources LOG: Going to deploy AWS resources
2025-03-18 12:46:35,243 [INFO] USER:anand_yadav deployment_processor.py:540:create_deployment_resources LOG: AWS resources were deployed successfully
2025-03-18 12:46:35,243 [INFO] USER:anand_yadav deployment_processor.py:543:create_deployment_resources LOG: Going to apply dynamic changes
2025-03-18 12:46:35,243 [INFO] USER:anand_yadav deployment_processor.py:545:create_deployment_resources LOG: Dynamic changes were applied successfully
2025-03-18 12:46:35,244 [INFO] USER:anand_yadav deployment_processor.py:547:create_deployment_resources LOG: Going to apply post deployment tags
2025-03-18 12:46:35,247 [INFO] USER:anand_yadav group_tagging_api_resource.py:135:apply_tags LOG: Tags were successfully applied
2025-03-18 12:46:35,247 [INFO] USER:anand_yadav deployment_processor.py:550:create_deployment_resources LOG: Going to create deploy output
2025-03-18 12:46:35,744 [INFO] USER:anand_yadav bundle_processor.py:71:create_deploy_output LOG: Output file with name task11_250318.124622/outputs/task11.json has been created
2025-03-18 12:46:35,744 [INFO] USER:anand_yadav deployment_processor.py:560:create_deployment_resources LOG: Deploy output for task11 was created.
2025-03-18 12:46:35,744 [INFO] USER:anand_yadav handlers.py:347:deploy LOG: Backend resources were deployed.
2025-03-18 12:46:35,744 [INFO] USER:anand_yadav helper.py:330:timed LOG: Stage deploy, elapsed time: 0:00:02.495749
2025-03-18 12:46:36,082 [INFO] USER:anand_yadav project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-18 12:46:36,082 [INFO] USER:anand_yadav project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-18 12:46:36,414 [INFO] USER:anand_yadav sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
